Spec.md

```diff
@@ -237,40 +237,53 @@ The comparison behavior of the package management system with respect to the
 `mod_version` is described below. The `mod_version` portion of the version
 number is mandatory.
 
-While the CKAN will accept *any* string as a `mod_version`, mod authors are
-encouraged to restrict version names to ASCII-letters, ASCII-digits, and the characters `.` `+` `-` `_`
-(full stop, plus, dash, underscore) and should start with a digit.
+While the CKAN will accept *any* ASCII string as a `mod_version`, mod authors
+are encouraged to restrict version names to letters, digits, and the
+ characters `.` `+` `-` `_` (full stop, plus, dash, underscore) and should
+ start with a digit.
 
 ###### Version ordering
 
-When comparing two version numbers, first the `epoch` of each are compared, then
-the `mod_version` if `epoch` is equal. `epoch` is compared numerically. The
-`mod_version` part is compared by the package management system using the
-following algorithm:
-
-The strings are compared from left to right.
-
-First the initial part of each string consisting entirely of non-digit
-characters is determined. These two parts (one of which may be empty) are
-compared lexically. If a difference is found it is returned. The lexical
-comparison is a comparison of ASCII values modified so that all the letters sort
-earlier than all the non-letters.
-
-Then the initial part of the remainder of each string which consists entirely of
-digit characters is determined. The numerical values of these two parts are
-compared, and any difference found is returned as the result of the
-comparison. For these purposes an empty string (which can only occur at the end
-of one or both version strings being compared) counts as zero.
-
-These two steps (comparing and removing initial non-digit strings and initial
-digit strings) are repeated until a difference is found or both strings are
-exhausted.
-
-Note that the purpose of epochs is to allow us to leave behind mistakes in
-version numbering, and to cope with situations where the version numbering
-scheme changes. It is not intended to cope with version numbers containing
-strings of letters which the package management system cannot interpret (such as
-ALPHA or pre-), or with silly orderings.
+When comparing two version numbers, first the `epoch` (continous integers at
```

Comment:
The format of an epoch is already defined above in the `###### epoch` section. Repeating it here is redundant, which is bad for readability and maintainability.